docs/operating-scylla/rest.rst

```diff
@@ -2,7 +2,35 @@
 REST
 ----
 
-For each JMX operation, attribute get and set, Scylla exposes a matching REST API. You can interact with the REST API using :code:`curl` or using the Swagger UI available at :code:`your-ip:10000/ui`
+ScyllaDB exposes a REST API to retrieve administrative information from a node and execute 
+administrative operations. For example, it allows you to check or update configuration, 
+retrieve cluster-level information, and more.
 
 
+CLI for REST API
+^^^^^^^^^^^^^^^^^
 
+ScyllaDB ships with ``scylla-api-client``, a lightweight tool that provides a command-line interface 
+for the ScyllaDB API. The tool allows you to:
+
+* List the API functions with their parameters and print detailed help for each function.
+* Invoke the API commands using the command line. 
+
+When invoking a function, ``scylla-api-client`` performs basic validation of the function arguments and 
+prints the result to the standard output. You can use commonly available command line utilities
+to print the results in the JSON format.
+
+To avoid errors, you should prefer ``scylla-api-client`` over ``curl`` and similar HTTP tools for interacting 
+with the ScyllaDB REST API.
+
+Usage
+======
+
+Run ``scylla-api-client --help`` for information about all the available options.
+
+**Examples:**
+
+* ``scylla-api-client --list-modules`` - Shows all the API modules.
+* ``scylla-api-client --list-module-commands system`` - Shows all the API commands for the ``system`` module.
+* ``scylla-api-client system uptime_ms`` - Gets the system uptime (in milliseconds).
+* ``scylla-api-client system log POST --message "hello world" --level warn`` - Sets the logger level.
```

Comment:
This command doesn't set the logger level, but rather:
```
$ scylla-api-client system log -h
POST - Write a message to the Scylla log

usage: system log POST --message MESSAGE --level {error,warn,info,debug,trace}

Required arguments:
  --message MESSAGE             The message to write to the log
  --level {error,warn,info,debug,trace}
                                The logging level to use
```

Reply:
Fixed. Please review.